sos/plugins/ipa.py

```diff
@@ -59,7 +59,7 @@ def retrieve_pki_logs(self, ipa_version):
                "/var/log/pki/pki-tomcat/ca/system",
                "/var/log/pki/pki-tomcat/ca/transactions",
                "/var/log/pki/pki-tomcat/catalina.*",
-               "/var/log/pki/pki-ca-spawn.*"
+               "/var/log/pki/pki-ca-spawn.*",
```

Comment:
This typo has been recently fixed.

Also in genral it makes sense to have one logical change (adding ArchPlugin to many plugins, e.g.) in one commit and an independent fix like this one in another.